Libraries/LibWeb/Parser/HTMLTokenizer.cpp

```diff
@@ -280,6 +300,20 @@ Optional<HTMLToken> HTMLTokenizer::next_token()
                 {
                     SWITCH_TO_AND_EMIT_CURRENT_TOKEN(Data);
                 }
+                ON_ASCII_UPPER_ALPHA
+                {
+                    m_current_token.m_tag.tag_name.append(tolower(current_input_character.value()));
```

Comment:
If a state does not emit, switch or reconsume in another state, it must end with `continue;` since otherwise we'll fall through into the `ANYTHING_ELSE` section.

Reply:
Whoops, overlooked that - fixed now.